secator/tasks/ai.py

```
@@ -837,5 +1277,6 @@ def _prompt_and_redetect(self, choices):
 		# Token breakdown for prompt display
 		by_role = self.history.count_tokens_by_role(self.model)
 		extra_data = {"tokens": by_role["total"], "context_window": get_context_window(self.model), "by_role": by_role}
-		items.append(Ai(content=answer, ai_type="prompt", extra_data=extra_data))
+		items.append(Ai(content=answer, ai_type="prompt", extra_data=extra_data,
+		                 message={"role": "user", "content": maybe_encrypt(answer, self.encryptor)}))
```

Flake8 (7.3.0)

[error] indentation contains mixed spaces and tabs

(E101)